example/src/App.js

```diff
@@ -1,14 +1,32 @@
 import React from "react";
 
 import DrawerLinks from "./DrawerLinks";
-import { AppDrawer, Dhis2, configureI18N } from "@blsq/blsq-report-components";
-
+import {
+  AppDrawer,
+  Dhis2,
+  configureI18N,
+  PluginRegistry
+} from "@blsq/blsq-report-components";
 import Invoices from "./invoices/Invoices";
 
 import customRoute from "./custom/CustomRoute";
 
 import { I18nextProvider } from "react-i18next";
 
+const Demo = props => {
+  return <span>complete {props.invoice.period} &nbsp;</span>;
+};
+const Demo2 = props => <span>validate</span>;
+
+const appPlugin = {
+  key: "exampleApp",
+  extensions: {
+    "invoices.actions": [Demo, Demo2]
+  }
+};
+
+PluginRegistry.register(appPlugin);
```

Comment:
I’m wondering if using something like https://github.com/smooth-code/react-teleporter wouldn’t be cleaner.
My problem with the current approach is that I would not allow to pass an array. If you want multiple things together, I’d make a separate component that you can organise as you want. Main concern is that the array does not give you any indication on how the component will render them